test/FluentMigrator.Tests/Unit/Generators/GenericGenerator/QuoterTests.cs

```diff
@@ -74,74 +74,74 @@ public override string ToString()
         [Test]
```

Comment:
I think this would be a lot cleaner with an NUnit Theory, so that you can parameterize the options values and outcomes.

In other words, while it might be a bit silly to set EnumAsString in the below CanEscapeAString test, it would then be possible to test not just a string, but an enum-as-string.

Thoughts?

Reply:
Unlike xUnit, NUnit has global Theory parameters (it might become painful to have multiple Theories in one class). Do you think tests for `QuoteColumnName` should be extracted to another class?
I am sorry, but I don't quite follow you with regards to `CanEscapeAString`. The property `EnumAsString` (now EnumAsUnderlyingType) does not affect `QuoteColumnName` which is tested by `CanEscapeAString`

Reply:
That's the point- the theory would verify it doesn't affect QuoteColumnName.  It might be needless complexity since an enum field name can't have a common escape in it - but Oracle and DB2 allow for arbitrary escapes, so it gets interesting fast.